src/replacer.js

```diff
@@ -82,21 +82,27 @@ export default class Replacer {
 					if (to !== undefined || this.replace) {
 						content = content.replaceAll(from, simpleTo ? to : (...args) => {
```

Comment:
Here is why I get an error concerning `data`: Here, when `simpleTo` is `true` and `to` is a function, it's passed as the second argument to `replaceAll` without a context, so `this` inside `to` is `undefined`.

It seems to me that we need to check the type of `to` earlier and take action accordingly.

Reply:
Yup, this is it. I tested with this change, and it worked:

<img width="799" alt="image" src="https://github.com/user-attachments/assets/bd60d8a2-d3a8-4288-a839-90a6a9192675">

<img width="579" alt="image" src="https://github.com/user-attachments/assets/86afede6-9f3b-447d-9946-95d1cbea1008">

Reply:
Ah good catch. Let me push a fix

Reply:
I'm not sure, but some checks in the code probably won't be needed at all. Do I get it right that if `to` is a function, the body of the arrow function `(...args) => {...}` won't be executed at all?

Reply:
Don't answer it. Your code shows the answer clearly :)